### DISCUSSION TITLE
Crossing data in different requests

### DISCUSSION THREAD

**Discussion by victorLessa** [adonisjs/core#2889] [Help]
Good night, I need some help.
I have a route that generates an xlsx with N operational data from the company where I work. When I export 1 spreadsheet everything goes well, the problem arises when I send two requests almost at the same time to this route, the result of the two requests is merged, information from request A in request B. Has anyone gone through this? Do you have any way of the stones to understand where I could have gone wrong in the code?

**Comment by feniljariwala82** [comment]
Can you please share your route code? That would be easy for me to understand your problem.
